tools/internal_ci/linux/grpc_psm_performance_gke_experiment.sh

```diff
@@ -0,0 +1,148 @@
+        -s driver_pool="${DRIVER_POOL}" -s driver_image= \
+        -s client_pool="${pool}" -s server_pool="${pool}" \
+        -s big_query_table="${table}" -s timeout_seconds=900 \
+        -s prebuilt_image_prefix="${PREBUILT_IMAGE_PREFIX}" \
+        -s prebuilt_image_tag="${UNIQUE_IDENTIFIER}" \
+        -s psm_image_prefix="${PSM_IMAGE_PREFIX}" \
+        -s psm_image_tag="${PSM_IMAGE_TAG}" \
+        -a ci_buildNumber="${KOKORO_BUILD_NUMBER}" \
+        -a ci_buildUrl="${CLOUD_LOGGING_URL}" \
+        -a ci_jobName="${KOKORO_JOB_NAME}" \
+        -a ci_gitCommit="${GRPC_GITREF}" \
+        -a ci_gitCommit_java="${GRPC_JAVA_GITREF}" \
+        -a ci_gitActualCommit="${KOKORO_GIT_COMMIT}" \
+        --prefix="${LOAD_TEST_PREFIX}" -u "${UNIQUE_IDENTIFIER}" -u "${pool}" -u "${uniquifier}"\
+        -a pool="${pool}" \
+        --category=psm \
+        --allow_client_language=c++ --allow_server_language=c++ \
+        -o "psm_${uniquifier}_loadtest_with_prebuilt_workers_${pool}.yaml"
+}
+
+buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "${PSM_PROXIED_TEST}"  --client_channels=8 --server_threads=16 --offered_loads 5000 6000 -l c++ -l java -l python
+buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "${PSM_PROXYLESS_TEST}" --client_channels=8 --server_threads=16 --offered_loads 5000 6000 -l c++ -l java -l python
+buildConfigs "${WORKER_POOL_32CORE}" "${BIGQUERY_TABLE_32CORE}" "${PSM_PROXIED_TEST}" --client_channels=32 --server_threads=64 --offered_loads 5000 6000 -l c++ -l java -l python
+buildConfigs "${WORKER_POOL_32CORE}" "${BIGQUERY_TABLE_32CORE}" "${PSM_PROXYLESS_TEST}" --client_channels=32 --server_threads=64 --offered_loads 5000 6000 -l c++ -l java -l python
+
+# Delete prebuilt images on exit.
+deleteImages() {
+    echo "deleting images on exit"
+    ../test-infra/bin/delete_prebuilt_workers \
+    -p "${PREBUILT_IMAGE_PREFIX}" \
+    -t "${UNIQUE_IDENTIFIER}"
+}
+trap deleteImages EXIT
+
+# Build and push prebuilt images for running tests.
+time ../test-infra/bin/prepare_prebuilt_workers \
+    -l "cxx:${GRPC_CORE_GITREF}" \
+    -l "java:${GRPC_JAVA_GITREF}" \
+    -l "python:${GRPC_CORE_GITREF}" \
+    -p "${PREBUILT_IMAGE_PREFIX}" \
+    -t "${UNIQUE_IDENTIFIER}" \
+    -r "${ROOT_DIRECTORY_OF_DOCKERFILES}"
+
+# Run tests.
+time ../test-infra/bin/runner \
+    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
+    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
+    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
+    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
+    -log-url-prefix "${LOG_URL_PREFIX}" \
+    -polling-interval 5s \
+    -delete-successful-tests \
+    -c "${WORKER_POOL_8CORE}:2" -c "${WORKER_POOL_32CORE}:2" \
```

Comment:
Here the queues should be set separately for proxied and proxyless.

Do we need 8-core and 32-core? If we do, we end up with four queues. I would make each of them have a concurrency level of one.